Makefile

```diff
@@ -72,7 +72,8 @@ transpile:
 redis: /usr/local/var/run/redis.pid
 	@echo
 	@echo "> Running Redis server..."
-	@redis-server &>/tmp/redis.log &
+	@if [ -z "$$REDIS_PASSWORD" ]; then REQUIREPASS="--requirepass=$$REDIS_PASSWORD"; fi
```

Comment:
nice ☺️  - I also had to remove the pidfile dependency in my machine as for some reason it was complaining - probably the pidfile location is very machine-dependant. Although I like the idea of starting redis only if it's not already running.

Reply:
Yeah, I didn't change that because I knew you had already changed it.